src/integrationTest/sam.test.ts

```diff
@@ -397,246 +397,206 @@ describe('SAM Integration Tests', async function () {
         console.log(`DebugSessions seen in this run:\n${sessionReport}`)
     })
 
+    describe('SAM Application creation errors', async () => {
+        let runtimeTestRoot: string
+
+        const randomTestScenario: TestScenario = {
+            runtime: 'nodejs14.x',
+            displayName: 'nodejs14.x (ZIP)',
+            path: 'hello-world/app.js',
+            debugSessionType: 'pwa-node',
+            language: 'javascript',
+            dependencyManager: 'npm',
+            vscodeMinimum: '1.50.0',
+        }
```

Comment:
We need to use a known valid scenario in an invalid situation, otherwise the test may pass for the wrong reasons. Choosing a scenario we're already testing is a decent way to achieve that.
```suggestion
        const randomTestScenario = scenarios[0]
```